hack/e2e.sh

```diff
@@ -30,6 +30,8 @@ METERING_SHORT_TESTS="${METERING_SHORT_TESTS:=false}"
 METERING_REPO_PATH="${METERING_REPO_PATH:=${ROOT_DIR}}"
 METERING_REPO_VERSION="$(basename $(find $MANIFESTS_BASE_DIR -type d | sort -r | head -n 1))"
 METERING_RUN_TESTS_LOCALLY="${METERING_RUN_TESTS_LOCALLY:=false}"
+METERING_OLM_SUBSCRIPTION_CHANNEL="${METERING_OLM_SUBSCRIPTION_CHANNEL:="4.6"}"
+IMAGE_METERING_ANSIBLE_OPERATOR_REGISTRY="${IMAGE_METERING_ANSIBLE_OPERATOR_REGISTRY:=""}"
```

Comment:
This is the name of the image that CI should be exposing once we update the release repository to start building the Dockerfile.registry. Else, you would need to manually override this variable when running the `make e2e` target:

```bash
$ ./hack/push-olm-manifests.sh quay.io/tflannag/metering-registry:latest
$ make e2e IMAGE_METERING_ANSIBLE_OPERATOR_REGISTRY=quay.io/tflannag/metering-registry:latest
```

Reply:
Made some more changes - renamed that variable and exported it in the `hack/lib/tests.sh` so you would only need to run `make e2e` with the `export METERING_ANSIBLE_OPERATOR_IMAGE_REGISTRY` specified locally (unless this is running in CI where the value we pull out of `$IMAGE_FORMAT` would take precedence.)